godel_robots/abb/godel_irb2400/godel_irb2400_support/urdf/abb_blending_eff_macro.xacro

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  
+  <xacro:include filename="$(find godel_robot_resources)/urdf/kinect2_sensor_macro.xacro"/>
+  <xacro:include filename="$(find godel_robot_resources)/urdf/calib_kinect2_sensor_macro.xacro"/>
+
+  <!-- parameter list -->
+  <xacro:property name="blend_tcp_x" value="-0.18415"/> 
+  <xacro:property name="blend_tcp_y" value="-0.088502"/>
+  <xacro:property name="blend_tcp_z" value="0.056356"/>
+
+  <xacro:property name="keyence_optical_x" value="0.287"/>
+  <xacro:property name="keyence_optical_y" value="-0.057"/>
+  <xacro:property name="keyence_optical_z" value="0.088502"/>
+
+  <xacro:property name="kinect_mount_x" value="0.09"/>
+  <xacro:property name="kinect_mount_y" value="-0.065"/>
+  <xacro:property name="kinect_mount_z" value="0.15"/>
+
+  <xacro:macro name="blending_eff_definition" params="prefix calibration">
+    <!-- Base link - attach your robot to this link -->
+    <link name="${prefix}eff_frame"/>
+
+    <link name="${prefix}tcp_frame"/>
+
+    <link name="${prefix}keyence_sensor_optical_frame"/>
+    <link name="${prefix}keyence_tcp_frame" />
+
+    <link name="${prefix}gripper_body">
+      <visual>
+        <geometry>
+          <mesh filename="package://godel_robot_config/meshes/visual/blending_eff.stl" />
```

Comment:
This mesh is complicated and ill-formed, so launching it as collision geometry will make MoveIt choke every time it loads a model. I need to move the model's from this directory into either this repo or the shared resources one.